src/main/java/net/sky/utils/StringUtils.java

```diff
@@ -0,0 +1,13 @@
+package net.sky.utils;
+
+public class StringUtils {
+
+    public static final String NEWLINE = System.getProperty("line.separator");
+
+    private StringUtils() {
+    }
+
+    public static String appendNewLine(String line) {
+        return line + NEWLINE;
```

Comment:
위 `NEWLINE` 을 제거하고 아래와 같이 작성하는게 더 나아 보이는데 어떠세요?
```suggestion
        return line + System.getProperty("line.separator");

Reply:
넵. 수정했습니다!